Utilities/ocvalidate/Checklist.md

```diff
@@ -0,0 +1,65 @@
+ocvalidate Checklist
+=====================
+
+As of commit <TODO>, ocvalidate performs the following checks:
+
+## ACPI
+- [ ] Add->Entry[N]->Path: Only `0-9, A-Z, a-z, '_', '-', '.', '/', and '\'` are accepted.
+- [ ] Add->Entry[N]->Path: `.dsl` filename suffix is not accepted.
+- [ ] Add->Entry[N]->Path: If a customised DSDT is added, remind to enable RebaseRegions in Quirks.
+- [ ] Add->Entry[N]->Comment: Only ASCII printable characters are accepted.
+- [ ] Delete->Entry[N]->Comment: Only ASCII printable characters are accepted.
+- [ ] Delete->Entry[N]->OemTableId/TableSignature: No size check for these two, as serialisation kills it.
+- [ ] Patch->Entry[N]->Comment: Only ASCII printable characters are accepted.
+- [ ] Patch->Entry[N]->OemTableId/TableSignature: No size check for these two, as serialisation kills it.
+- [ ] Patch->Entry[N]->Find/Replace/Mask/ReplaceMask: Identical size of these patterns guaranteed.
```

Comment:
Similar to Comment. This thing is global for many patches. Put it to global section.

Reply:
Yes, done. But one exception is under `Kernel->Patch`: When `Base` is used, `Find` can be empty, and thus its size can be different from `Replace` and masking sections.